tests/__init__.py

```diff
@@ -1,9 +1,29 @@
+import inspect
 from contextlib import asynccontextmanager
-
+import pytest
 from scrapy import Request
 from scrapy.http.response.html import HtmlResponse
 from scrapy.utils.test import get_crawler
 
+from scrapy_playwright.handler import windows_get_result
+
+
+def windows_pytest_mark_asyncio(pytest_mark_asyncio):
```

Comment:
I suspect this will not work after #252, is there another pytest hook that could be leveraged?

Reply:
If you remove pytest-asyncio, you still need to add some wrapper to every test function, as Windows needs a special process.